.gitlab-ci.yml

```diff
@@ -486,6 +486,13 @@ vmware.sh_ova:
 
 filesystem.sh:
   extends: .integration
+  parallel:
+    matrix:
+    - RUNNER:
+      # don't execute on fedora-39 yet
+      - aws/fedora-38-x86_64
```

Comment:
Would you mind expanding the comment to say why it's not run there yet? Maybe something like: 
```
# don't execute on fedora-39 yet until https://lkml.kernel.org/linux-xfs/20230823220225.3591135-1-david@fromorbit.com/#t is in the kernel
# see also https://github.com/osbuild/osbuild-composer/pull/3820#issuecomment-1881587335 for more details
```